modules/mapper-extras/src/main/java/org/elasticsearch/index/query/RankFeatureQueryBuilder.java

```diff
@@ -240,10 +241,55 @@ void doXContent(XContentBuilder builder) throws IOException {
             }
 
             @Override
-            Query toQuery(String field, String feature, boolean positiveScoreImpact) throws IOException {
+            Query toQuery(String field, String feature, boolean positiveScoreImpact) {
                 return FeatureField.newSigmoidQuery(field, feature, DEFAULT_BOOST, pivot, exp);
             }
         }
+
+        /**
+         * A scoring function that scores documents as {@code indexed value of S}
```

Comment:
Small comment, it feels confusing to render "indexed value of S" as code, maybe this could be "as simply {@code S} itself, where ..."